apps/web/src/pages/templates/editor/TemplateEditorPage.tsx

```diff
@@ -84,6 +97,20 @@ export default function TemplateEditorPage() {
               {activePage === 'Add' && (
                 <AddChannelsPage channelButtons={channelButtons} handleAddChannel={handleAddChannel} />
               )}
+              {loadingEditTemplate
+                ? null
+                : activePage === 'in_app' &&
+                  inAppFields.map((message, index) => {
+                    return (
+                      <TemplateInAppEditor
+                        disabled={!activeChannels[ChannelTypeEnum.IN_APP]}
+                        key={index}
+                        errors={errors}
+                        control={control}
+                        index={index}
+                      />
+                    );
+                  })}
```

Comment:
```suggestion
              {!loadingEditTemplate && activePage === 'in_app' ?
                  inAppFields.map((message, index) => {
                    return (
                      <TemplateInAppEditor
                        disabled={!activeChannels[ChannelTypeEnum.IN_APP]}
                        key={index}
                        errors={errors}
                        control={control}
                        index={index}
                      />
                    ) : null
                  })}
```

Maybe something like that?

Reply:
oh right!!! 
I was thinking of all the other pages which will also depend on loadingEditTemplate